src/pages/basic_data/institution_admin/AddEmployeePage.tsx

```diff
@@ -59,19 +61,27 @@ export const AddEmployeePage = () => {
     }
 
     let personId = values.user.id;
+    const nationalId = values.user.nationalId;
+    const { nvi, ...personWithoutNvi } = values.user;
 
     if (!personId) {
+      const person = nationalId ? personWithoutNvi : values.user;
       // Create Person if it does not yet exist in Cristin
       const cristinPerson: CreateCristinPerson = convertToCristinPerson({
-        ...values.user,
+        ...person,
         employments: [values.affiliation],
       });
       const createPersonResponse = await createCristinPerson(cristinPerson);
       if (isErrorStatus(createPersonResponse.status)) {
         dispatch(setNotification({ message: t('feedback.error.create_user'), variant: 'error' }));
       } else if (isSuccessStatus(createPersonResponse.status)) {
-        personId = createPersonResponse.data.id;
-        await new Promise((resolve) => setTimeout(resolve, 10_000)); // Wait 10sec before creating NVA User. TODO: NP-9121
+        if (!nationalId) {
+          dispatch(setNotification({ message: t('feedback.success.create_person'), variant: 'success' }));
+          resetForm();
```

Comment:
Om man legger til ansettelse for en person uten fødselsnummer så nullstilles ikke ansettelsesfelt etter submit success